frontend/src/providers/AppProviders.tsx

```diff
@@ -300,7 +314,11 @@ export const AppProvider: React.FC<{ children: ReactNode }> = ({ children }) =>
             subscriptionTiers, // New state variable
             setSubscriptionTiers, // Setter for subscriptionTiers
             isFinancialAssistantActive,
-            setIsFinancialAssistantActive
+            setIsFinancialAssistantActive,
```

Comment:
@egdagger Where is the logic to set the financial agent as true when the parameter `agent` is equal to `financial`?